opi/plugins/jami.py

```diff
@@ -0,0 +1,42 @@
+import opi
+from opi.plugins import BasePlugin
+
+class Jami(BasePlugin):
+	main_query = "jami"
+	description = "Jami p2p messenger."
+	queries = [main_query, 'jami-qt', 'jami-gnome', 'jami-daemon']
+
+	@classmethod
+	def run(cls, query):
+		if not opi.ask_yes_or_no("Do you want to install jami from jami repository?", 'y'):
+			return
+
+		print("Which version do you want to install?")
+		option = opi.ask_for_option(options=[
+			Jami.main_query,
+			'jami-qt',
+			'jami-gnome',
+			'jami-daemon',
+		])
+
+		print("You have chosen %s", option)
+
+		if opi.get_version() == '15.4':
+			repourl = 'https://dl.jami.net/nightly/opensuse-leap_15.4/'
+		elif opi.get_version() == '15.3':
+			repourl = 'https://dl.jami.net/nightly/opensuse-leap_15.3/'
```

Comment:
```suggestion
		if opi.get_distribution().startswith('openSUSE:Leap'):
			repourl = 'https://dl.jami.net/nightly/opensuse-leap_%s/' % opi.get_version()
```

This way we don't need to update this plugin for every leap version